### PR TITLE
Feature/get rep by division

### PR DESCRIPTION
**Pull request by alistaire47**
Does a few things:

- Adds a script to `inst/` to read the OCD division ID data from GitHub, doing minimal transformation, and update it in the package using `usethis::use_data()`. This can just be rerun by maintainers in the future to update the data.
- Adds the data, exported as `ocd_divisions`
- Adds a function called `get_rep_by_division()` that can take a division and return representative info. Based on `get_rep_address()`, with a few tweaks.
- Various cleanups to make life easier and fix `R CMD check` issues, including
  - turning on markdown mode for roxygen
  - letting roxygen generate `NAMESPACE`
  - switching to `Authors@R`
  - formatting the license
  - fixing some whitespace and formatting issues
  - putting nonstandard files in `.Rbuildignore`
  - etc.

This PR also _causes_ an `R CMD check` warning, unfortunately, because some OCD division IDs include non-ASCII characters, e.g. `ocd-division/country:us/state:nm/county:doña_ana/precinct:75`. This could be avoided by encoding them with `stringi::stri_escape_unicode` before saving and then decoding with `stringi::stri_unescape_unicode()`, but this would add a dependency and may make grepping harder, which is annoying. Maybe there's a better solution here.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
+    of the Google Civic Information API.
+License: MIT + file LICENSE
 Imports: 
     httr, 
     jsonlite
 Depends: R (>= 2.10)
 Encoding: UTF-8
 LazyData: true
-RoxygenNote: 7.0.1
+Roxygen: list(markdown = TRUE)
```
Comment (alistaire47) [inline DESCRIPTION:16]: This makes writing docs way easier

File: LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# MIT License
```
Comment (alistaire47) [inline LICENSE.md:1]: Generated by `usethis::use_mit_license()`

File: R/get_rep_address.R [hunk 3]
```diff
@@ -16,39 +16,42 @@
 #'
 #' @examples
 #' \dontrun{
-#' get_rep_address(address = "55 e monroe, chicago, il",level="regional", role="schoolBoard", key = Sys.getenv("google_civic_api"))
+#' get_rep_address(
```
Comment (alistaire47) [inline R/get_rep_address.R:19]: `R CMD check` was complaining about this as a long line

File: LICENSE.md (hunk 2 above)
Comment (willdebras) [reply LICENSE.md:1]: Good call on adding a license!

**Comment by willdebras** [comment]
These all look great! Merging now. By the end of the day we can probably add news/changelog, run checks and see if there is anything standing in the way for CRAN submission.

**Pull request merged**